lib/domain.js

```diff
@@ -64,18 +82,38 @@ exports.create = exports.createDomain = function() {
 
 // the active domain is always the one that we're currently in.
 exports.active = null;
+Domain.prototype.members = undefined;
 
+Domain.prototype._enable = function() {
```

Comment:
can you make this function a top-level function? Exposing it inside the prototype does not seem correct, as an external user could call it.

Reply:
Updated